packages/quantic/force-app/main/default/lwc/quanticNotifications/quanticNotifications.css

```diff
@@ -1,7 +1,15 @@
+:host {
+  --slds-c-toast-sizing-min-width: 100%;
+}
+
 .notification {
   background-color: var(---lwc-colorBackgroundAlt, white);
   border: 1px solid var(--lwc-colorBackgroundToast, rgb(116, 116, 116));
   margin-right: 0;
   margin-left: 0;
   width: 100%;
+}
+
+.notification-container {
+  z-index: 20;
```

Comment:
z-index is not good with the expandable search box:

https://github.com/user-attachments/assets/98464d13-2fc8-480e-bae2-3af6246a4401

Reply:
nice catch, the notifications I had tested were under the tabs, so I did not catch this one. 

I changed the z-index from 20 to 2 since the quanticSearchbox is at 5.

This allows the expandable searchbox to be ok and the notifications to still take priority over regular page content which has z-index of 1

<img width="584" alt="image" src="https://github.com/user-attachments/assets/38226541-23ef-4f56-9453-b8b103121bae">